examples/distroless/test.bats

```diff
@@ -0,0 +1,17 @@
+true
+# shellcheck disable=SC2034
+CH_TEST_TAG=$ch_test_tag
+
```

Comment:
I think these four lines are not needed? They aren't used and the test title prints out properly.

Reply:
Every other `test.bats` I looked at has these lines, is this something that should by tidied up outside of this specific test?

Reply:
Probably. Some might actually need it, or maybe it's completely obsolete.

Reply:
I'll create a separate issue to track this.

Reply:
It looks like the `true` may not be needed. Without line 3 there is this error:  
```
 ✗ NO_TAG_SET/hello
   (in test file /home/heasterday/charliecloud/examples/distroless/test.bats, line 13)
     `[[ $status -eq 0 ]]' failed
   ch-run[657842]: can't find image: /var/tmp/images/NO_TAG_SET: No such file or directory (ch-run.c:146 2)
   ```  
And line 2 suppresses this shellcheck warning
```
   CH_TEST_TAG=$ch_test_tag
^---------^ SC2034: CH_TEST_TAG appears unused. Verify use (or export if used externally).
```

Reply:
Bizarre; it must be some `ch-test` magic I've forgotten about.

The `true` is needed: without it, the `shellcheck disable` applies to the whole file, not just the subsequent line.

Maybe these gotchas should be documented in the contributors' guide?

Reply:
Added back the `true`. Documenting the process for adding test does indeed sound like a good idea.